voc/virtual_machine.go

```diff
@@ -30,5 +30,6 @@ type VirtualMachine struct {
 	BlockStorage      []ResourceID       `json:"blockStorage"`
 	BootLogging       *BootLogging       `json:"bootLogging"`
 	MalwareProtection *MalwareProtection `json:"malwareProtection"`
-	OSLogging         *OSLogging         `json:"oSLogging"`
+	OSLogging         *OSLogging         `json:"osLogging"`
```

Comment:
Why do you want change the naming? In that case we need an extra hack for the owl2go generator.

Reply:
We decided to keep it and hack it in owl2go.